Public/js/forum-app.js

```diff
@@ -0,0 +1,176 @@
+$(document).ready(function() {
+  var Post = function() {
+    this.type;
+    this.message;
+    this.user;
+    this.date = new Date().toString().slice(0,21);
+  }
+
+  Post.prototype.makePost = function() {
+    $.ajax({
+      context: this,
+      type: 'POST',
+      url: 'reservations',
+      data: {
+        reservation: {
+          type: this.type,
+          message: this.message,
+          date: this.date
+        }
+      },
+      dataType: 'json',
+      success: function(response) {
+        if(response.ok) {
+          moveOn('reserve');
+        } else if(response.reservationExists) {
+          if(new Date(this.day).toString() == new Date(today.slice(0,15)).toString()) {
+            runError('This reservation is taken',0);
+          } else {
+            runError('This reservation is taken',1);
+          }
+        } else if(response.reservationLimit) {
+          if(new Date(this.day).toString() == new Date(today.slice(0,15)).toString()) {
+            runError('You have already used your two reservations today',0);
+          } else {
+            runError('You cannot book more than two spots per day',1);
+          }
+        } else if(!response.authenticated) {
+          logOut();
+        } else {
+          console.log(response);
+        }
+      }
+    });
+  };
+
+  var fillTable = function(response,table,hour) {
+    for(var a=hour;a<24;a++) {
```

Comment:
It would be nice to replace variables `a`, `b` and `c` with names that are more intuitive / semantic